File: java/32 LCM.java
package code;
import java.util.Scanner;
public class Test {
	static int LCM(int num1,int num2) {
		int min=num1>num2?num2:num1;
		int res=num1*num2;
		int hcf=1;
		for(int i=min;i>0;i--) {
			if(num1%i==0 && num2%i==0) {
				hcf=i;
				break;
			}
		}
		res/=hcf;
		return res;
	}
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter first nummber: ");
		int num1=sc.nextInt();
		System.out.print("Enter second nummber: ");
		int num2=sc.nextInt();
		System.out.println("LCM is "+LCM(num1,num2));
	}
}

File: python/32 LCM.py
def LCM(num1,num2):
    if num1>num2:
        min=num2
    else:
        min=num1
    res=num1*num2
    hcf=1
    for a in range(min,0,-1):
        if num1%a==0 and num2%a==0:
            hcf=a
            break
    res//=hcf
    return res
num1=int(input("Enter first number: "))
num2=int(input("Enter second number: "))
print(f"LCM is {LCM(num1,num2)}")
